### DISCUSSION TITLE
Raymond  Calls significant percentage of GCC current and future potential contributors - an autists

### DISCUSSION THREAD

**Discussion by AnimusPEXUS** [rms-support-letter/rms-support-letter.github.io#6813] [General]
There is undergoing discussion in GCC chat about --canceling-- blocking 'toxic' people from their mailing lists.
https://gcc.gnu.org/pipermail/gcc/2021-April/235575.html

>The choice to have a policy for ejecting jerks has serious costs.
One of those costs is the kind of rancorous dispute that has been
burning like a brushfire on this list the last few weeks.  Another,
particularly serious for hackers - is that such a policy is hostile to
autists and others who have poor interaction skills but can ship good
code.  This is a significant percentage of your current and future
potential contributors, enough that excluding them is a real problem.

Raymond then suggests silently ban people without any code of conduct

>You're better off with an informal system of moderator fiat and
*without* rules that beg to become a subject of dispute and
manipulation. A strong norm about off-list behavior and politics being
out of bounds here is also helpful.

Lenin applauds from the mausoleum!

**Comment by nukeop** [comment]
I actually follow the same approach, no rules and bans for people who are destructive for the project.

**Reply by AnimusPEXUS** [reply]
details matter.

it's not the banning mechanisms I wanted to highlite here, but attitude to the people.

'toxicity', 'justice', 'god', 'faith' - this and terms like this, are undetermined terms. Any time I see people using those terms instead/inplace of explaining their point of view and/or their actions, I'm automatically start thinking negatively of such people, like of cons/fakers.

You see: It is very easy today to call somebody 'toxic', and this will be enough to start anti-person mob, which doesn't even damn cares about what hides behind particular 'toxic' blaming.

**Reply by 6r1d** [reply]
> undetermined terms

Alrighty

> using those terms instead/inplace of explaining their point of view

You won't like a lot of people that way, but I see your point. I'd ask the "why" question if I were you. Many times, so the logic breaks. Then again, people change topics in that case.

> It is very easy today to call somebody 'toxic'

Yes, **don't wear the badges others give you**.

**Reply by AnimusPEXUS** [reply]
Also, note, what after Stallman's last statement, there is mood started to call him an autist. - this is incorrect, because:
1. psychology is questionable science
2. only specialist can give to somebody an authist diagnosis

just wanted to note, what probably E Raymond is somewhat incorrect, or maybe even somehow 'toxic' to 'significant percentage of GCC current and future potential contributors' :D

**Reply by 6r1d** [reply]
> what probably E Raymond is somewhat incorrect, or maybe even somehow 'toxic' to 'significant percentage of GCC current and future potential contributors' :D

*Cue the intro music!*

**Welcome to the wide and weird world of diagnosing yourself and your friends!**
If there's **any** minor inconvenience in communication and you don't like, just blame autism!

* Someone is tired and doesn't answer? Autism!
* Someone's reading a book?! It's 21st century and the reader is a poser, it's autism!
* Someone is really, really mad at you, but didn't start swearing yet? It's time to diagnose **terminal** autism (and maybe invent a couple of new words along the way)!

__It's you, it's someone else, it's everyone and you don't need to be a doctor__!

Alright, enough with my wrong opinion on modern culture… Pfft, no way.

Somehow, it became a trend everywhere around us and I'm glad it did not hit you yet. But beware and be prepared, it's everywhere now…
🍚🍱🍡Like the messages full of Unicode emojis!🍻🥃🍹

> psychology is questionable science

It doesn't really matter for people who don't know how to do sciency sciencing and don't want to know. They just want to hang badgess and labels on others.

-----

Now imagine how many times ESR was called an autist. Because I'm surprised you weren't. Lol.

**Comment by AnimusPEXUS** [comment]
RH laborer says he can't explain himself  - just as expected
https://gcc.gnu.org/pipermail/gcc/2021-April/235574.html: 
- Somebody: Stallman is normal human
- RH worker: Stallman is wrong and should be removed. I don't have enough energy to explain, but **trust me**

**Reply by nukeop** [reply]
Are you in any way related to Daniel Pocock?

**Reply by AnimusPEXUS** [reply]
No. I'm not related to anybody. I'm just simple guy not from USA.
